src/naming.adoc

```diff
@@ -76,11 +76,11 @@ of the P extension.
 
 === Additional Standard Unprivileged Extension Names
 
-Standard unprivileged extensions can also be named using a single "Z" followed by
-an alphabetical name and an optional version number. For example,
-"Zifencei" names the instruction-fetch fence extension described in
-<<zifencei>>; "Zifencei2" and
-"Zifencei2p0" name version 2.0 of same.
+Standard unprivileged extensions can also be named by using a single "Z" followed by an
+alphanumeric name. The name must end with an alphabetical character or an 
+optional version number. The second letter from the end cannot be numeric if the
+last letter is ``p''. For example, "Zifencei" names the instruction-fetch fence extension
```

Comment:
I'm not well versed in asciidoc. Does ``p'' mean something? Documentation I found shows ``p`` would be monospace, but I couldn't find anything for mixing `` and ''.

Reply:
I'm not sure what is up with that either! Glad you found and fixed it.